src/analyses/base.ml

```diff
@@ -178,14 +178,15 @@ struct
     | LNot -> ID.lognot
 
   let unop_FD = function
```

Comment:
There's also a copy in `BaseInvariant` apparently. Maybe these could be deduplicated in `BaseUtil`, `ValueDomain` or something?